xcp_d/utils/bids.py

```diff
@@ -361,6 +373,21 @@ def collect_run_data(layout, bold_file, cifti=False):
             suffix="xfm",
         )
 
+    elif not cifti:
+        run_data["boldref"] = layout.get_nearest(
+            bids_file.path,
+            strict=False,
+            suffix="boldref",
+        )
+        run_data["boldmask"] = layout.get(
+            return_type="file",
+            suffix="mask",
+            desc="brain",
+        )
+        run_data["t1w_to_native_xform"] = layout.get(
+            return_type="file", datatype="anat", suffix="xfm", to="MNI152NLin2009cAsym"
+        )
```

Comment:
Just noticed this. It looks like the T1w-to-native transform is set to be a transform from any anatomical space (which probably would just be the T1w space) to _MNI_ space, rather than native space.

Reply:
I'm guessing this isn't a real problem in practice, since the HCP "transform files" are just copies of its preprocessed T1w file, which is a separate thing we need to deal with.